src/code-templates/libraries/logger/logger.ts

```diff
@@ -1,10 +1,9 @@
-import { configurationProvider } from "configuration-provider";
+import {configurationProvider} from "../configuration-provider";
+import {PinoLogger} from './pino.logger';
+import {LOG_LEVELS} from "./definition";
 
-// example on how to read the level: configurationProvider.get("logger.level")
-export const info = (message) => {
-  console.log(message);
-};
-
-export const error = (message) => {
-  console.error(message);
-};
+export default new PinoLogger(
+    configurationProvider.get("logger.level") as LOG_LEVELS,
+    configurationProvider.get("logger.prettyPrint"),
```

Comment:
@henarbel I thought you wanted to decouple it from configuration lib, no? I'm good for now with both options